src/main/java/core/basesyntax/services/FileReader.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.services;
+
+import java.util.List;
+
+public interface FileReader {

```

Comment:
Better name it FileReaderService, so user programmer can easily know what he works with(other service classes too)